packages/react-i18n/src/i18n.ts

```diff
@@ -500,6 +510,27 @@ export class I18n {
     });
   }
 
+  private humanizeFutureDate(date: Date, options?: Intl.DateTimeFormatOptions) {
+    const time = this.formatDate(date, {
+      ...options,
+      hour: 'numeric',
+      minute: '2-digit',
+    }).toLocaleLowerCase();
+
+    if (isToday(date)) {
+      return time;
+    }
+
+    if (isTomorrow(date)) {
+      return this.translate('date.humanize.tomorrow', {time});
```

Comment:
This translation key already existed?

Reply:
My understanding is that the translations are provided by the consuming library (because I couldn't find `Yesterday at {time}' anywhere). Is there a place I should be putting translations?